local_docs/fluid.waveform~.maxref.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes'?>
+<?xml-stylesheet href="./_c74_ref.xsl" type="text/xsl"?>
+<!--
+Part of the Fluid Corpus Manipulation Project (http://www.flucoma.org/)
+Copyright 2017-2019 University of Huddersfield.
+Licensed under the BSD-3 License.
+See license.md file in the project root for full license information.
+This project has received funding from the European Research Council (ERC)
+under the European Union’s Horizon 2020 research and innovation programme
+(grant agreement No 725899).
+-->
+<c74object name='fluid.waveform~' category='FluidCorpusManuipulation'>
+    <digest>Visualise audio waveforms and data.</digest>
+    <description>A JSUI object for visualising waveforms, feature data, slices and spectrograms.</description>
+    <discussion>
+        <h4><openfilelink filename="Fluid Corpus Manipulation Toolkit.maxpat">Open the Overview Patch</openfilelink></h4>
+        The fluid.waveform~ object is designed for visualising audio waveforms, feature data, slices and spectrograms. This is a convenient way to interrogate analysis, or visualise the results of processes.
+
+        <p>fluid.waveform~ is part of the Fluid Decomposition Toolkit of the FluCoMa project. For more explanations, learning material, and discussions on its musicianly uses, visit <a href="http://www.flucoma.org/">flucoma.org</a>.</p>
+    </discussion>
+	<!--METADATA-->
+	<metadatalist>
+		<metadata name='author'>FluCoMa</metadata>
+		<metadata name='tag'>Fluid Decomposition</metadata>
+		<metadata name='tag'>corpus</metadata>
+	</metadatalist>
+
+    <inletlist>
+    </inletlist>
+
+    <outletlist>
+    </outletlist>
+
+    <objarglist>
+    </objarglist>
+
+    <!--MESSAGES-->
+    <methodlist>
+    <method name='waveform'>
+        <arglist>
+            <arg name='buffer' optional='0' type='symbol' />
+            <arg name='color' optional='1' type='symbol' />
+        </arglist>
+        <description>
+        Paint a waveform from a buffer.
+        
+        </description>
+    </method>
+    <method name='features'>
+        <arglist>
+            <arg name='buffer' optional='0' type='symbol' />
+            <arg name='color' optional='1' type='symbol' />
+        </arglist>
+        <description>
+        Paint a feature curve from a buffer.
+        
+        </description>
+    </method>
+    <method name='image'>
+        <arglist>
+            <arg name='buffer' optional='0' type='symbol' />
+        </arglist>
+        <description>
+        Construct an image from a two-dimensional buffer~ of values.
+        
+        </description>
+    </method>
+    <method name='slices'>
+        <arglist>
+            <arg name='markers buffer' optional='0' type='symbol' />
+            <arg name='reference' optional='0' type='symbol' />
+        </arglist>
+        <description>
+        Paint vertical slices stored in a buffer~ onto the waveform.
```

Comment:
Is here a good place to mention that they can be moved, added, deleted {and one day, written out!}

Reply:
Tackled in b290c6e